ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/patch-panel.j2

```diff
@@ -7,6 +7,27 @@
 {% if patch_panel is arista.avd.defined %}
 !
 patch panel
+{%     if patch_panel.connector is arista.avd.defined %}
+{%         set pp_connector_cli = "connector " %}
+{%         if patch_panel.connector.interface is arista.avd.defined %}
+{%             set pp_connector_cli = pp_connector_cli ~ "interface " %}
+{%             if patch_panel.connector.interface.recovery is arista.avd.defined %}
+{%                 set pp_connector_cli_recovery = pp_connector_cli ~ "recovery " %}
+{%                 if patch_panel.connector.interface.recovery.review_delay.min is arista.avd.defined and patch_panel.connector.interface.recovery.review_delay.max is arista.avd.defined %}
+{%                     set pp_connector_cli_recovery = pp_connector_cli_recovery ~ "review delay " ~ patch_panel.connector.interface.recovery.review_delay.min ~ " " ~ patch_panel.connector.interface.recovery.review_delay.max %}
```

Comment:
Same here you can just check for the required delay min/max and maybe just print it directly with `   connector interface recovery review delay {{ foo }} {{ bar }}`